fuel-base/src/test/kotlin/fuel/HttpLoaderBuilderTest.kt

```diff
@@ -12,7 +12,8 @@ import org.junit.Test
 import java.net.SocketTimeoutException
 import java.util.concurrent.TimeUnit
 
-class HttpLoaderBuilderTest {
+internal class HttpLoaderBuilderTest {
```

Comment:
Without a specific accessible modifier for explicit API mode, Kotlin compiler doesn't allow it.